.github/workflows/release.yaml

```
@@ -42,12 +45,14 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Push Charts to GHCR
+      - name: Push Charts to GHCR and Sign
         run: |
           shopt -s nullglob
           for pkg in .cr-release-packages/*; do
             if [ -z "${pkg:-}" ]; then
               break
             fi
-            helm push "${pkg}" oci://ghcr.io/${GITHUB_REPOSITORY_OWNER}/charts
+            repo="oci://ghcr.io/${GITHUB_REPOSITORY_OWNER}/charts"
+            helm push "${pkg}" "${repo}" |& grep Digest: | awk '{print $NF}' > digest.txt
+            cosign sign --yes "${repo}/oauth2-proxy@$(cat digest.txt)"
```

```suggestion
            cosign sign --yes "${repo}/${pkg}@$(cat digest.txt)"
```

shouldn't this be referencing the pkg variable?

---

or does package contain the relative filesystem path? in that case we should do something like:

pkg=$(basename "${pkg%/}")

---

Fixed, thanks!

---

Fixed again, thanks!